src/main/python/systemds/operator/operation_node.py

```diff
@@ -45,27 +45,25 @@ class OperationNode(DAGNode):
     _output_types: Optional[Iterable[VALID_INPUT_TYPES]]
     _source_node: Optional["DAGNode"]
     _brackets: bool
+    _datatype_is_unknown: bool
 
     def __init__(self, sds_context: 'SystemDSContext', operation: str,
                  unnamed_input_nodes: Union[str,
                                             Iterable[VALID_INPUT_TYPES]] = None,
                  named_input_nodes: Dict[str, VALID_INPUT_TYPES] = None,
-                 output_type: OutputType = OutputType.MATRIX,
                  is_python_local_data: bool = False,
-                 brackets: bool = False):
+                 brackets: bool = False,
```

Comment:
I added two new boolean attributes to operation_node: datatype_is_none and datatype_is_unknown to handle operations with no return value (write, print,..) or unkown datatype (list access), which where previously indicated by the OutputType.NONE